Automattic/pocket-casts-ios
# Profile bookmark sync sort option

Fixes #

Ensure that profile bookmarks sort option is synced between devices.

## To test

- You need to use the staging environment
- You need to enable the FF settings_sync and restart the app on each device/simulator
- Start the app on two different devices with the same account that needs to have Plus active
- Go to Profile -> Bookmarks
- Set a sort option
- Go back to the profile screen and pull to refresh
- On the other device pull to refresh
- Open Profile -> Bookmarks
- Ensure that the sort option active is the same as in the other device.

## Checklist
